docs/index.rst

```diff
@@ -3,20 +3,20 @@
 
 .. _index_wsgrb:
 
-Wattsight API python library documentation
+Volue Insight API Python library documentation
 ==========================================
 
-:Maintainer: <support@wattsight.com>
-:Organization: `Wattsight AS`_
+:Maintainer: <support@volueinsight.com>
```

Comment:
Should we use support.insight@volue.com instead? Who reads the mails sent to support@volueinsight.com?

Reply:
yes, definitely! thanks for the reminder @stigalvestad  👍